log/log.go

```diff
@@ -54,15 +54,25 @@ type CommonLogger interface {
 
 // ControlLogger provides methods to config a logger.
 type ControlLogger interface {
```

Comment:
Not a fan of `ControlLogger` but can't think of anything else to name this 😆

Reply:
@coderabbitai give good suggestions for this interface by thinking about how to use it

Reply:
@gaby @efectn what do you think about the given names ?

Reply:
`ConfigurableLogger` is our choice